web/src/lib/components/album-page/albums-list.svelte

```diff
@@ -267,9 +267,15 @@
   };
 
   const handleDeleteAlbum = async (albumToDelete: AlbumResponseDto) => {
-    await deleteAlbum({
-      id: albumToDelete.id,
-    });
+    try {
+      await deleteAlbum({
+        id: albumToDelete.id,
+      });
+    } catch {
+      // In rare cases deleting an album completes after the list of albums has been requested,
+      // leading to a bad request error.
+      // Since the album is already deleted, the error is ignored.
+    }
```

Comment:
I woud rethrow the error in all other cases

```ts
const isBadRequest = isHttpError(error) && error.status === 400;
if (!isBadRequest) {
  throw error;
}
 ```

Reply:
I wanted just wanted to ask this..., thanks.